.vscode/scripts/run-vehicledatabroker.sh

```diff
@@ -20,47 +20,45 @@ echo "#######################################################"
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-DATABROKER_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.version | tr -d '"')
-DATABROKER_PORT='55555'
-DATABROKER_GRPC_PORT='52001'
-sudo chown $(whoami) $HOME
-DATABROKER_ASSET_FOLDER="$ROOT_DIRECTORY/.vscode/scripts/assets/databroker/$DATABROKER_VERSION"
+DATABROKER_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.image | tr -d '"')
+DATABROKER_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.version | tr -d '"')
```

Comment:
same, here, should we use app-manifest?

Reply:
See comments to prerequisite_settings.json